plasmapy/physics/dielectric.py

```diff
@@ -196,3 +199,40 @@ def cold_plasma_permittivity_LRP(B, species, n, omega):
         R += - omega_p ** 2 / (omega * (omega + omega_c))
         P += - omega_p ** 2 / omega ** 2
     return L, R, P
+
+# plasma permittivity in terms of Jackson function
+@u.quantity_input(omega=u.rad /u.s,
+                  kWave=u.rad /u.m,
+                  T=u.K,
+                  n=u.m**-3,
+                  z_mean=u.dimensionless_unscaled)
+def permittivity_1D_Maxwellian(omega,
+                               kWave,
+                               T,
+                               n,
+                               particle,
+                               z_mean=None):
+    """
+    omega [rad/s]
+    kWave [rad/m]
+    temperature [eV]
+    density [cm^-3]
+    """
+    vTh = parameters.thermal_speed(T=T,
+                                   particle=particle,
+                                   method="most_probable")
+    # plasma frequency
+    wp = parameters.plasma_frequency(n=n,
+                                     particle=particle,
+                                     z_mean=z_mean)
+    # scattering parameter alpha.
+    # explicitly removing factor of sqrt(2) to be consistent with Froula
+    alpha = np.sqrt(2) * (wp / (kWave * vTh)).to(u.dimensionless_unscaled)
+    zeta = (omega / (kWave * vTh)).to(u.dimensionless_unscaled)
+    # correction for comparing this 1D Maxwellian based definition
+    # to a 3D Maxwellian based permittivity
+    # this is sqrt(pi) / sqrt(pi) ** 3
+    #    corr = 1 / np.pi
+    corr = 1
+    chi = corr * alpha ** 2 * jackson_function(zeta.value)
```

Comment:
Is `corr` included here to make the derivation/logic cleaner? Do you think it'd make a good default (`=1`) argument?

Reply:
I basically copy pasted this from my own code and corr here was a correction (I think for 1D vs. 3D integrals) that involved a factor of pi (as seen in the comment above) `corr = 1`. I think this line can just be removed.